src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -1825,6 +1824,24 @@ namespace DataZoneEquipment {
                 break;
             }
         }
+        // Set MinAirLoopIterationsAfterFirst for equipment that uses sequenced loads, based on zone equip load distribution scheme
+        int minIterations = DataHVACGlobals::MinAirLoopIterationsAfterFirst;
+        if (this->LoadDistScheme == DataZoneEquipment::LoadDist::SequentialLoading) {
+            // Sequential needs one extra iterations up to the highest airterminal unit equipment number
+            minIterations = max(coolingPriority, heatingPriority, minIterations);
+        }
+        else if (this->LoadDistScheme == DataZoneEquipment::LoadDist::UniformLoading) {
+            // Uniform needs one extra iteration which is the default
+        }
+        else if (this->LoadDistScheme == DataZoneEquipment::LoadDist::UniformPLRLoading) {
+            // UniformPLR needs two extra iterations, regardless of unit equipment number
+            minIterations = max(2, minIterations);
+        }
+        else if (this->LoadDistScheme == DataZoneEquipment::LoadDist::SequentialUniformPLRLoading) {
+            // SequentialUniformPLR needs one extra iterations up to the highest airterminal unit equipment number plus one more
+            minIterations = max((coolingPriority + 1), (heatingPriority + 1), minIterations);
+        }
+        DataHVACGlobals::MinAirLoopIterationsAfterFirst = minIterations;
```

Comment:
Was this logic culled from somewhere else?  Or is this new?  And if it's new, is there a justification for the iteration count logic?